__tests__/SimpleMutation.test.ts

```diff
@@ -0,0 +1,78 @@
+import { Reactor, App, Triggers, Args, Timer, OutPort, InPort, TimeUnit, TimeValue, Origin, Log, LogLevel, Action } from '../src/core/internal';
+
+/* Set a port in startup to get thing going */
+class Starter extends Reactor {
+    public out = new OutPort<number>(this);
+
+    constructor(parent: Reactor|null) {
+        super(parent);
+        this.addReaction(
+            new Triggers(this.startup),
+            new Args(this.writable(this.out)),
+            function(this, __out) {
+                __out.set(4);
+
+            }
+        );
+    }
+
+}
+
+class R1 extends Reactor {
+    public in = new InPort<number>(this);
+    public out = new OutPort<number>(this);
+
+    constructor(parent: Reactor|null) {
+        super(parent);
+        this.addReaction(
+            new Triggers(this.in),
+            new Args(this.in, this.writable(this.out)),
+            function(this, __in, __out) {
+                __out.set(4)
+            }
+        )
+    }
+
+
+
+}
+
+class R2 extends Reactor {
+    public in = new InPort<number>(this);
+    public out = new OutPort<number>(this);
+
+    constructor(parent: Reactor|null) {
+        super(parent);
+        this.addMutation(
+            new Triggers(this.in),
+            new Args(this.in, this.out),
+            function(this, __in, __out) {
+                test('expect error to be thrown', () => { 
+                    expect(() => {
+                        this.connect(__out, __in)
+                    }).toThrowError("New connection introduces direct feed through.")
```

Comment:
The error reported here should be that `__in` is already occupied.

Reply:
Tagging @Wonseo-C

Reply:
https://github.com/lf-lang/reactor-ts/pull/98/commits/62b751a020f21a46b3eb8c5ef7d17f0912e9b7e3
I tried to fix the error message for this test case by adding the logic for the error in the ```canConnect``` function.